### PR TITLE
In IE link ui dialog doesn't appear

### PR DESCRIPTION
**Pull request by wrench13**
In IE link UI dialog doesn't appear because it use `window.scrollX` and `window.scrollX` that doesn't work in IE.

I determine what to use in this case with conditions on page offset support.

### CODE REVIEW

**Comment by anthonyjb** [comment]
@wrench13 thanks again for these, quick question which version of IE is this an issue in? Is it all of them?

**Comment by anthonyjb** [comment]
BTW I ask because I'm currently considering dropping support for 9 as it's such a small user base now and most of the workarounds in the code are for IE9. I suspect this is a IE10  problem too reading some posts online. 

What are your thoughts on IE9 support, do you think it's still worth it, or is it better to move to IE10 as the base IE browser?

**Comment by guillaumepiot** [comment]
I confirm I saw the same issue on IE10, link tooltip doesn't show when testing on client windows laptop. I told them to use Chrome for now.

**Comment by anthonyjb** [comment]
What do you think about dropping IE9 support @guillaumepiot?

**Comment by guillaumepiot** [comment]
I think supporting for IE9 is now obsolete since there's currently 0.4% of users according to W3 stats. 
[http://www.w3schools.com/browsers/browsers_explorer.asp](http://www.w3schools.com/browsers/browsers_explorer.asp)

Also considering that the tools will be mostly used by website administrators, I would always advised the clients to run the latest version of their favourite browsers.

I think we should leave the current bug fix in place for IE and focus on IE10 and above for further enhancement. It would then be up the developer to add backward compatibility on a project basis.

**Comment by guillaumepiot** [comment]
From looking at a broad audience website analytics, IE 10 & 11 constitute about 85% of the IE traffic. IE traffic being only 10% of total traffic.

**Comment by wrench13** [comment]
@anthonyjb I had same issue on IE9, 10 and 11 on windows.
@guillaumepiot I agree with you on IE9 support.

**Comment by anthonyjb** [comment]
OK excellent so I think as part of release 1.4 (so after the next minor release) we'll look to drop support for IE9 so that we can remove some of the workarounds currently in place for things like CSS class management.

I'm away for a couple of days now but once I'm back I'll merge this and any other IE bug fixes in and they can go out in the next patch release (1.2.3).

**Comment by anthonyjb** [comment]
Thanks again @wrench13 :+

File: src/scripts/ui/dialogs/link.coffee [hunk 1]
```diff
@@ -101,7 +101,8 @@ class ContentTools.LinkDialog extends ContentTools.AnchoredDialogUI
         # Input
         @_domInput.addEventListener 'keypress', (ev) =>
             if ev.keyCode is 13
-                @save()
+              ev.preventDefault()
```
Comment (anthonyjb) [inline src/scripts/ui/dialogs/link.coffee]: @wrench13 can I ask why this line was removed?

**Comment by anthonyjb** [comment]
Hi @wrench13 - in the end I've used your pull request to write a general function for getting the scroll position cross browser (just figure it's bound to be useful for other tools). Thanks again for your help here.

**Comment by wrench13** [comment]
Sounds great ! I'm away from my computer for few weeks, sorry for my silence on your question.

**Pull request closed without merging**